_config.yml

```diff
@@ -87,5 +87,5 @@ exclude:
   - css
   - image-templates
   - CODE_OF_CONDUCT.md
+  - Staticfile
   - package-lock.json
-  - assets/uswds-theme
```

Comment:
just to double-check: it's ok to no longer exclude `assets/uswds-theme`?

Reply:
Correct! I added our brand colors to the theme directory in a previous PR.